zebra/if_netlink.c

```diff
@@ -267,8 +267,8 @@ static void netlink_determine_zebra_iftype(const char *kind,
 		*zif_type = ZEBRA_IF_VETH;
 	else if (strcmp(kind, "bond") == 0)
 		*zif_type = ZEBRA_IF_BOND;
-	else if (strcmp(kind, "bond_slave") == 0)
-		*zif_type = ZEBRA_IF_BOND_SLAVE;
+	else if (strcmp(kind, "bond_member") == 0)
```

Comment:
This breaks what the kernel is sending us right?  this has to stay `bond_slave` since it is data from the kernel